src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -57,11 +85,20 @@ void setUserStore(UserStore userStore) {
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
 
-    // TODO(rafer45): Investigate security implications of using Session
     String username = (String) request.getSession().getAttribute("user");
     User user = userStore.getUser(username);
 
+    int userCount = userStore.Count();
```

Comment:
super nit: move these inside the if statement since that is the scope they are used in.

Reply:
Acknowledged. (will fix in next PR)